src/ui/public/notify/toasts/toast_notifications.js

```diff
@@ -0,0 +1,20 @@
+export class ToastNotifications {
+  constructor() {
+    this.list = [];
+    this.idCounter = 0;
+  }
+
+  add = toast => {
```

Comment:
Just wondering, do you think it would be nice to expose an api more resembling the old notify one? Keep `add()`, but also include `success()`, `warning()`, etc? Those would just call add with the appropriate color.

Reply:
Good suggestion guys, will do!